rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -13,8 +13,18 @@
 
 
     private final static int OPEN_SUCCESS = 1;
+    private final static int TIMEOUT = 10;
     private long rtmpPointer = 0;
 
+    /* Timeout value in seconds */
+    private int timeout = TIMEOUT;
+    /* Sets the timeout variable */
+    public void setTimeout(int timeout) {
```

Comment:
Nit: I'm guessing that this method should probably go below the static exception class (e.g. just above the open method).

Please also add proper javadoc (e.g. /** */ rather than /* */, and @param documenting the argument - including units).

Reply:
Done.